### PR TITLE
Introduce kubevirt provider

### PR DESCRIPTION
**Pull request by pkliczewski**
We update provider dependencies to enable [kubevirt](http://www.kubevirt.io/) provider.

### CODE REVIEW

**Comment by geemus** [comment]
Looks like fog-kubevirt hasn't been released yet? Also, if you add me as an administrator on your repo, I can move it into the fog org and get that part sorted out for you. I'd like to see both of those things done before we merge further things. Thanks!

**Comment by pkliczewski** [comment]
@geemus I transferred the repository to you, I am not able to give admin to my personal org.

**Comment by geemus** [comment]
@pkliczewski ah cool, yeah I probably gave slightly incorrect instructions (it's been a while since I've initiated a transfer). I think that should work though, will let you know if I need anything else. Thanks!

**Comment by geemus** [comment]
@pkliczewski I think it should be transferred and you are added as admin. I only added you and @plribeiro3000 but I think as admin you can add others if you need or want to. Just let me know if you need anything else on that front. Thanks!

File: lib/fog/kubevirt.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+require 'fog/kubevirt/compute'
```
Comment (masayag) [inline lib/fog/kubevirt.rb]: this should be modified to `fog/compute/kubevirt'` to match https://github.com/fog/fog-kubevirt/blob/master/lib/fog/compute/kubevirt.rb

**Comment by pkliczewski** [comment]
We still want to continue to work on it. Can we have it reopen or should I create new PR?

**Comment by geemus** [comment]
Sorry, I had thought we resolved this from the discussion, but my memory is faulty at times. Reopening is fine. Thanks!

**Pull request closed without merging**